webapp/store/auth.js

```diff
@@ -46,6 +46,9 @@ export const getters = {
   termsAndConditionsAgreed(state) {
     return state.user && state.user.termsAndConditionsAgreedVersion === VERSION
   },
+  locale(state) {
+    return state.locale
```

Comment:
should we have a fallback here @ogerly ? most of the getters I have seen in or store have `|| null`, etc...

Reply:
@ogerly I think you can remove this. You're consuming `this.user.locale` so this code here is never used.